haystack/nodes/answer_generator/openai.py

```diff
@@ -26,11 +27,12 @@ def __init__(
         self,
         api_key: str,
         model: str = "text-curie-001",
-        max_tokens: int = 7,
-        top_k: int = 5,
-        temperature: int = 0,
+        max_tokens: int = 20,
+        top_k: int = 2,
+        temperature: float = 0.2,
         presence_penalty: float = -2.0,
         frequency_penalty: float = -2.0,
+        max_calls_per_minute: int = 60,
```

Comment:
Shouldn't we make the default here 3000? I'd prefer having here the "production" setting for a paid OpenAI account. For people on the free trial we can make the error message that you added further down more descriptive.